### DISCUSSION TITLE
How to reduce the font size of the complication?

### DISCUSSION THREAD

**Discussion by UberQueiroz** [j-kaltes/Juggluco#228] [Q&A]
Hi!

Is there a way to reduce the font size of the watch complication? I'd like to make mine more discrete, if possible.

Thanks a lot!

![Screenshot_20241120_230951_glucodata](https://github.com/user-attachments/assets/294c5105-c6f8-4df0-aeab-f5d26ac20446)

**Comment by j-kaltes** [comment]
The only way is to use a watch face with a smaller small image complication slot.

EDIT:
To make it more discreet, you can also use the Value+Arrow complication and choose less visible colors.

**Comment by j-kaltes** [comment]
The text of the complication in Juggluco 9.0.1 can be made smaller: https://www.juggluco.nl/Juggluco/download.html
![config](https://github.com/user-attachments/assets/6259510a-40ba-4207-9e17-502235c4d483)
![smaller](https://github.com/user-attachments/assets/a00a299e-2e01-466f-a8a2-8bdad5e752b1)
